packages/grafana-ui/src/components/Button/Button.tsx

```diff
@@ -60,10 +62,27 @@ export const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
       iconOnly: !children,
     });
 
+    const buttonStyles = cx(
+      styles.button,
+      {
+        [css(styles.disabled, {
+          '&:hover': css(styles.disabled),
+        })]: disabled,
+      },
```

Comment:
Why isn't this a part of the existing button styles?

I feel like line ~240 could be extended to 

```ts
disabled: css(disabledStyles, {
  "&:hover": disabledStyles,
}),
```